swap/tests/testutils/mod.rs

```diff
@@ -62,7 +62,7 @@ impl BobParams {
     }
 }
 
-pub struct BobEventLoopJoinHandle(JoinHandle<()>);
+pub struct BobEventLoopJoinHandle(pub JoinHandle<()>);
```

Comment:
Instead of making this public, why no forward the `abort` call? newtypes are pretty useless if their inner fields are public.